src/common/util/include/openvino/util/file_util.hpp

```diff
@@ -239,5 +240,20 @@ inline std::basic_string<C> make_plugin_library_name(const std::basic_string<C>&
            FileTraits<C>::library_ext();
 }
 
+/**
+ * @brief load binary data from file
+ * @param mutex - mutex
+ * @param path - binary file path to load
+ * @return binary vector
+ */
+std::vector<uint8_t> load_binary(std::mutex& mutex, const std::string& path);
```

Comment:
I think mutex should be taken outside of this function.

Reply:
I'll remove mutex arg from api.